dvc/tree/http.py

```diff
@@ -33,7 +33,7 @@ class HTTPTree(BaseTree):  # pylint:disable=abstract-method
 
     SESSION_RETRIES = 5
     SESSION_BACKOFF_FACTOR = 0.1
-    REQUEST_TIMEOUT = 10
+    REQUEST_TIMEOUT = 60
```

Comment:
This might also be not enough if the file is big enough and/or the connection is slow enough. What is the biggest file size that you are uploading? We are having a similar discussion about webdav where apparently chunked upload is not always possible and so that makes https://github.com/iterative/dvc/issues/829 more relevant. Maybe it does here too.

Reply:
The chunk size has never caused a problem for a single large file in my case. Only having too many concurrent files uploading would make some of them time out. 
In general, with a slow enough connection, or too many concurrent jobs, some file will eventually fail to upload. 
This relates to other issues discussing the implementation of retries.